Library/Formula/gtk-mac-integration.rb

```diff
@@ -0,0 +1,162 @@
+require 'formula'
+
+class GtkMacIntegration < Formula
+  homepage 'https://wiki.gnome.org/Projects/GTK%2B/OSX'
+  url      'http://ftp.gnome.org/pub/gnome/sources/gtk-mac-integration/2.0/gtk-mac-integration-2.0.7.tar.xz'
+  sha256   'e6de70da740b452f18c113a7ebb6772936a0fb9873137514090849820e296eb7'
+  head     'git://git.gnome.org/gtk-mac-integration'
+
+  depends_on 'pkg-config' => :build
+  depends_on 'xz' => :build
+  depends_on 'cairo' => 'without-x11'
+  depends_on 'pango' => 'without-x11'
+  depends_on 'gtk+' => 'without-x11'
+  depends_on 'pygtk' => :optional
+
+  head do
+    depends_on 'automake' => :build
+    depends_on 'autoconf' => :build
+    depends_on 'libtool' => :build
+    depends_on 'gtk-doc' => :build
+  end
+
+  stable do
+    patch :DATA
```

Comment:
If upstream has committed this already, can we use upstream's commit directly instead of having to inline the patch? The syntax here can be ` patch :DATA if build.stable? ` as well :).